examples/pokedex/src/api.js

```diff
@@ -0,0 +1,95 @@
+import { gql } from "apollo-boost";
+import { ApolloClient } from "apollo-client";
+import { createHttpLink } from "apollo-link-http";
+import { InMemoryCache } from "apollo-cache-inmemory";
+
+const httpLink = createHttpLink({
+  uri: "https://graphql-pokemon.now.sh/"
+});
+
+const client = new ApolloClient({
+  link: httpLink,
+  cache: new InMemoryCache()
+});
+
+export const fetchList = async () => {
+  let data;
+
+  await client
+    .query({
+      query: gql`
+        {
+          pokemons(first: 10000) {
+            id
+            name
+            image
+            classification
+            types
+          }
+        }
+      `
+    })
+    .then(result => {
+      data = result.data.pokemons;
+    });
+
+  return data;
+};
+
+export const fetchPokemon = async (id) => {
+  let data;
+
+  await client
```

Comment:
Do the same cleanup as above (This is an example code so I am extra picky about code styling, sorry 😄)